en/sensor/optical_flow.md

```diff
@@ -32,7 +31,7 @@ An exemplary setup is the [PX4Flow](../sensor/px4flow.md) and [Lidar-Lite](../se
 
 #### PX4Flow
 
-The easiest way to calculate the optical flow is to use the [PX4Flow](../sensor/px4flow.md) board (see link for setup information).
+Currently PX4 supports both the [PX4Flow](../sensor/px4flow.md) and the [Bitcraze Flow](../sensor/bitcraze_flow.md) (see link for setup information). 
```

Comment:
This doesn't work because it is under a "PX4Flow" heading. Split out into two as below. What you want here is the same info as intro to the device - a basic overview of what it is, and ideally what separates it from the alternatives. Succinct though, because most info should be in the linked docs.

```
#### PX4Flow

[PX4Flow](../sensor/px4flow.md) is an optical flow smart camera that has a native resolution of 752x480 pixels and calculates optical flow on a 4x binned and cropped area at 400 Hz, giving it a very high light sensitivity (see link for setup information).

#### Bitcraze Flow

[Bitcraze Flow](../sensor/bitcraze_flow.md) is a BLAH BLAH (see link for setup information). 
```